src/containers/SearchPage/searchHelpers.js

```diff
@@ -294,12 +294,15 @@ const mapResourcesToItems = (resources, ltiData, isLti, t) =>
       ? getLtiUrl(resource.path, resource.id)
       : resource.contexts?.length
       ? getContextUrl(resource.contexts[0])
-      : resource.path,
+      : plainUrl(resource.path),
```

Comment:
Denne må med for å unngå problemet her: https://staging.ndla.no/search?query=hvordan%20ta%20blodtrykk. Url til første emne er til article-api.